install_config/topics/manually_configuring_for_vsphere.adoc

```diff
@@ -0,0 +1,190 @@
+
+[Disk]
+        scsicontrollertype = pvscsi <14>
+
+[Network]
+        public-network = "VM Network" <15>
+----
+<1> Any properties set in the `[Global]` section are used for all specified vcenters unless overriden by the settings in the individual `[VirtualCenter]` sections.
+<2> vCenter username for the vSphere cloud provider.
+<3> vCenter password for the specified user.
+<4> Optional. Port number for the vCenter server. Defaults to port `443`.
+<5>  Set to `1` if the vCenter uses a self-signed cert.
+<6> Name of the data center on which Node VMs are deployed.
+<7> Override specific `[Global]` properties for this Virtual Center. Possible setting scan be `[Port]`, `[user]`, `[insecure-flag]`, `[datacenters]`. Any settings not specified are pulled from the `[Global]` section.
+<8> Optional. Additional vCenter server.
+<9> Set any properties used for various vSphere Cloud Provider functionality. For example, dynamic provisioning, Storage Profile Based Volume provisioning, and others.
+<10> IP Address or FQDN for the vCenter server.
+<11> Path to the VM directory for node VMs.
+<12> Set to the name of the datastore to use for provisioning volumes using the storage classes or dynamic provisioning. Prior to 3.9, if the datastore is located in a storage directory or is a member of a datastore cluster, the full path was required.
+<13> Optional. Set to the path to the resource pool where dummy VMs for Storage Profile Based volume provisioning should be created.
+<14> Type of SCSI controller the VMDK will be attached to the VM as.
+<15> Set to the network port group for vSphere to access the node, which is called VM Network by default. This is the node host's ExternalIP that is registered with Kubernetes.
+
+The cluster installation process configures single-zone or single vCenter by default.
+
+Deploying {product-title} in vSphere on different zones can be helpful to avoid
+single-point-of-failures, but creates the need for shared storage across zones.
+If an {product-title} node host goes down in zone "A" and the pods
+should be moved to zone "B".
+See https://kubernetes.io/docs/admin/multiple-zones/#limitations[Multiple zone
+limitations] in the Kubernetes documentation for more information.
+
+[IMPORTANT]
+====
+This ensures that the VMDK always presents a consistent UUID to the VM, allowing the disk
+to be mounted properly.
+
+====
++
+For every virtual machine node that will be participating in the cluster:
+VM properties -> VM Options -> Advanced -> Configuration Parameters -> disk.enabledUUID=TRUE
++
+Alternatively, link:https://github.com/vmware/govmomi/tree/master/govc[GOVC tool] can be used:
++
+.. Set up the GOVC environment:
++
+[source,bash]
+----
+export GOVC_URL='vCenter IP OR FQDN'
+export GOVC_USERNAME='vCenter User'
+export GOVC_PASSWORD='vCenter Password'
+export GOVC_INSECURE=1
+----
++
+. Find the Node VM paths:
++
+[source,bash]
+----
+govc ls /datacenter/vm/<vm-folder-name>
+----
+.. Set disk.EnableUUID to true for all VMs:
```

Comment:
This says to set it to true, but below it is set to `1`. Does it matter? And we seem to use "TRUE" in other places. Can it be either "TRUE" or "1"?

Reply:
It could be. I believe I just pulled the verbage from vmware's upstream docs here:
https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html